README.md

```diff
@@ -96,20 +102,25 @@ rpcuser=xxxxxx
 rpcpassword=xxxxxx
 
 rpcport=18232
-rpcallowip=0.0.0.0
+rpcallowip=127.0.0.1
 ```
+Reset your working directory to your `zingolib/reg/`
 
-In another terminal instance, navigate to your `lightwalletd` directory, and run
-`./lightwalletd --no-tls-very-insecure --zcash-conf-path ~/.zcash/zd/zcash.conf --data-dir ~/.zcash/zd/ --log-file ~/.zcash/ld/lwd-log --log-level 7`
+`zcashd` works in regtest with the following invocation (with explicit pathes - you will have to adjust to your own path):
+`./zcashd --printtoconsole -conf=/zingolib/.zc/zd/zcash.conf --datadir=/zingolib/.zc/zd/ -debuglogfile=/zingolib/.zc/zd/debug.log -debug=1`
+
+In another terminal instance, working from the same `zingolib/reg/` directory run:
+`./lightwalletd --no-tls-very-insecure --zcash-conf-path ./.zc/zd/zcash.conf --data-dir ./.zc/zd/ --log-file ./.zc/ld/lwd-log --log-level 7`
 `lightwalletd`'s terminal output will not have clear success, but the log file will show something like:
 `{"app":"lightwalletd","level":"info","msg":"Got sapling height 1 block height 0 chain regtest ..."}`
+which you can view with `tail -f` or your favorite tool.
 
-Now you will need to add blocks to your regtest chain if you have not done so previously.
-In still another terminal instance in the `zcash` directory, you can run `./src/zcash-cli -regtest generate 11` to generate 11 blocks.
+You will need to add blocks to your regtest chain if you have not done so previously.
+In still another terminal instance in the `zingolib/reg/` directory, you can run `.zcash-cli -regtest generate 11` to generate 11 blocks.
```

Comment:
`./zcash-cli` with the slash instead of `.zcash-cli` without?

Reply:
Correct.